game_of_life.c

```diff
@@ -60,6 +60,28 @@ struct options start_menu()
     return current_options;
 }
 
+/**
+* Checks if the Input is valid
+**/
+int validate_input(struct options start_menu)
+{
+    int validate = 0;   // Validate 0 = Input is not Valid
+                        // Validate 1 = Input is Valid
+
+    if ((start_menu.height <= 2) || (start_menu.width <= 2)) {
+        printf("The input of the height or width is incorrect.\n");
+        validate = 0;
+    } else {
+        validate = 1;
+    }
+
+    if (validate = 0) {
+        return 0;
+    } else {
+        return 1;
+    }
```

Comment:
Wenn du abhängig des wertes returnst, kannst du doch auch sofort den wert returnen.
```suggestion
    return validate;
```